src/NJsonSchema.CodeGeneration/DefaultEnumNameGenerator.cs

```diff
@@ -24,11 +24,47 @@ public class DefaultEnumNameGenerator : IEnumNameGenerator
         /// <returns>The enumeration name.</returns>
         public string Generate(int index, string name, object value, JsonSchema schema)
         {
+
             if (string.IsNullOrEmpty(name))
             {
                 return "Empty";
             }
 
+            switch (name)
+            {
+                case ("="):
+                    name = "eq";
```

Comment:
I think we should make this upper case so that "-Foo" gets converted to "MinusFoo" => the default output should be Upper Camel Case, please add some tests so showcase the change => there should already be some tests somewhere in the project.

Reply:
Added test coverage and changed output to Camel Case.